### PR TITLE
Feature: Set up Routing & FAQ Component

### PR DESCRIPTION
**Pull request by ssimmons05**
# [Set up routing](https://trello.com/c/HVSJHQ84/19-create-router)

## A short description of what this PR does.
Create routes for each of the pages.

## How was it tested?
This was linted with Yarn and tested with Jest.
router_test_screenshot

### Yarn Lint
router_lint_screenshot

### Checklist
- [x] I acknowledge that all my contributions will be made under the project's license
- [x] I have made a material change to the repo (functionality, testing, spelling, grammar)
- [x] I have read the [Contribution Guidelines](https://github.com/aaronshivers/snapit-hacktoberfest/blob/main/CONTRIBUTING.md) and my PR follows them
- [x] I have titled the PR appropriately
- [x] I have updated my branch with the main branch
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added the necessary documentation about the functionality in the appropriate .md file
- [x] I have added inline documentation to the code I modified

### CODE REVIEW

File: src/router/Router.jsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+import { BrowserRouter, Route, Switch } from "react-router-dom";
+import NavBar from "../components/NavBar";
+import Information from "../Pages/Information";
+import Profile from "../Pages/Profile";
+import FAQ from "../Pages/FAQ";
+import Registration from "../Pages/Registration";
+
+const Router = () => (
+  <BrowserRouter basename="/HacktoberFest">
+    <div className="App">
```
Comment (dylhack) [inline src/router/Router.jsx:11]: Should we add NavBar to the className here? It would allow us to style the entire navbar across the screen like so:
```css
.NavBar {
  background-color: white;
  min-width: 100%;
}
```

**Review by dylhack — CHANGES_REQUESTED** [review]

File: src/App.test.jsx [hunk 2]
```diff
@@ -1,12 +1,65 @@
 import React from "react";
-import { shallow } from "enzyme";
+import Enzyme, { shallow } from "enzyme";
+import EnzymeAdapter from "enzyme-adapter-react-16";
 import App from "./App";
+import FAQ from "./Pages/FAQ";
+import Information from "./Pages/Information";
+import Profile from "./Pages/Profile";
+import Registration from "./Pages/Registration";
```
[the remark this answers is not in the document]
Comment (ssimmons05) [reply src/App.test.jsx]: I fixed this issue here: e75b67f215c546217f36ccdf88d7fc1bf3b65061

[the remark this answers is not in the document]
Comment (dylhack) [reply src/App.test.jsx]: Loosk good

**Comment by dylhack** [comment]
@aaronshivers and/or @seccabimmons could you check some of the conversations I started on this code? I'm not quite sure how they should be approached or if they will be fixed in the future with another PR.

File: src/App.css [hunk 3]
```diff
@@ -1,43 +1,32 @@
...
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
-}
-
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+.FAQ {
```
Comment (aaronshivers) [inline src/App.css:9]: :wrench: Everything related to the FAQ page needs to be moved to a separate PR. We want to keep our PR's small and for one thing at a time.

**Review by aaronshivers — CHANGES_REQUESTED** [review]
:wrench: Everything related to the FAQ page needs to be moved to a separate PR. We want to keep our PR's small and for one thing at a time.

**Comment by aaronshivers** [comment]
@ssimmons05 
You don't have to close this PR, just change it to a draft, create a new branch from this one, then remove the appropriate code from each branch.

**Comment by dylhack** [comment]
I closed it, it would be better not to waste commits just to revert them
